keyword-analyzer/src/test/java/com/example/keywordanalyzer/repository/DocumentCollectionRepositoryTest.java

```diff
@@ -0,0 +1,78 @@
+package com.example.keywordanalyzer.repository;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.example.keywordanalyzer.model.constants.AnalysisStatus;
+import com.example.keywordanalyzer.model.constants.CollectStatus;
+import com.example.keywordanalyzer.model.entity.DocumentCollection;
+
+@SpringBootTest
+@Transactional
+public class DocumentCollectionRepositoryTest extends BasicRepositoryTest<DocumentCollection> {
+	@Autowired
+	public DocumentCollectionRepositoryTest(DocumentCollectionRepository repository) {
+		super(repository);
+	}
+
+	@Override
+	protected Long getId(DocumentCollection entity) {
+		return entity.getDocumentCollectionId();
+	}
+
+	@Override
+	protected DocumentCollection createMockData() {
+		int collectStatus = CollectStatus.NOT_STARTED;
+		int analysisStatus = AnalysisStatus.NOT_STARTED;
+		LocalDateTime startTime = LocalDateTime.of(2024, 1, 1, 0, 0, 0);
+		LocalDateTime endTime = LocalDateTime.of(2024, 1, 1, 1, 0, 0);
+		int totalDocumentCount = 0;
+		String message = "test";
+		return new DocumentCollection(collectStatus, analysisStatus, startTime, endTime, totalDocumentCount, message);
+	}
+
+	@Override
+	protected void assertDataEquals(DocumentCollection expected, DocumentCollection actual) {
+		assertEquals(expected.getCollectStatus(), actual.getCollectStatus());
+		assertEquals(expected.getAnalysisStatus(), actual.getAnalysisStatus());
+		assertEquals(expected.getStartTime(), actual.getStartTime());
+		assertEquals(expected.getEndTime(), actual.getEndTime());
+		assertEquals(expected.getTotalDocumentCount(), actual.getTotalDocumentCount());
+		assertEquals(expected.getMessage(), actual.getMessage());
```

Comment:
```suggestion
assertAll(
    () -> assertEquals(expected.getCollectStatus(), actual.getCollectStatus()),
    () -> assertEquals(expected.getAnalysisStatus(), actual.getAnalysisStatus()),
    () -> assertEquals(expected.getStartTime(), actual.getStartTime()),
    () -> assertEquals(expected.getEndTime(), actual.getEndTime()),
    () -> assertEquals(expected.getTotalDocumentCount(), actual.getTotalDocumentCount()),
    () -> assertEquals(expected.getMessage(), actual.getMessage())
);
```
- 가독성 있게 assertAll 활용

Reply:
assertAll 적용 완료